docs/application-connector/docs/08-04-register-secured-api.md

```diff
@@ -82,6 +82,8 @@ This is an example of the `api` section of the request body for an API secured w
 
 When you register an API with the `credentials.certificateGen` object, the Application Registry generates a SHA256withRSA-encrypted certificate and a matching key. To enable communication between Kyma and an API secured with this authentication method, set the certificate as a valid authentication medium for all calls coming from Kyma in your external solution.
 
+>**NOTE:** You can retrieve client certificate by sending a request to get details of a registered service: ``` curl https://gateway.{CLUSTER_DOMAIN}/{APP_NAME}/v1/metadata/services/{YOUR_SERVICE_ID} --cert {CERT_FILE_NAME}.crt --key {KEY_FILE_NAME}.key -k``` Successful call will return response body with base64 encoded client certificate.
```

Comment:
You can retrieve the client certificate by sending the following request:
``` 
curl https://gateway.{CLUSTER_DOMAIN}/{APP_NAME}/v1/metadata/services/{YOUR_SERVICE_ID} --cert {CERT_FILE_NAME}.crt --key {KEY_FILE_NAME}.key -k
``` 
A successful call will return a response body with the details of a registered service and a base64-encoded client certificate.

-----
Maybe we can get place it outside the Note, to have a proper codeblock? Or it needs to be a note?

Reply:
O.K, I moved it outside the note.